### ISSUE TITLE
Errors building Gruntfile.js

### ISSUE THREAD

**Issue by alkhuhsm** [maenu/grunt-template-jasmine-istanbul#32]
Hi I seem to be having a problem running the gruntfile.js

This is what I get

root@dev-NW504:/var/www# grunt --config Gruntfile.js --force
Running "jshint:all" (jshint) task
>> 0 files linted. Please check your ignored files.

Running "connect:default_options" (connect) task
Started connect web server on http://0.0.0.0:8000

Running "jasmine:tests" (jasmine) task
Testing jasmine specs via phantom
>> ReferenceError: Can't find variable: __coverage__ at
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/grunt-template-jasmine-istanbul/reporter.js:17 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:1788 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:2160 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:2144 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:2106 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:2049 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:2145 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine.js:802 
>> http:/127.0.0.1:8000/.grunt/grunt-contrib-jasmine/jasmine-helper.js:23 

Warning: PhantomJS timed out, possibly due to an unfinished async spec. Used --force, continuing.

=========================================================

I'm pretty new to all this and its the first time I actually use grunt so I will need you to please explain the problem in layman terms.

**Comment by gyandeeps** [comment]
@alkhuhsm is this still an issue?

**Comment by gyandeeps** [comment]
Closing, please feel free to reopen if this issue still persists.
